recipe/meta.yaml

```diff
@@ -9,24 +9,38 @@ source:
   sha256: 502ad98707319f4a51fa2ca1c677bd659008d27ded9f6380c79e8932e38dcdf2
 
 build:
-  number: 1
-  script: python setup.py install
+  number: 2
   skip: True  # [py>=35]
 
 requirements:
-  build:
-    - python
-  run:
+  host:
     - python
+    - pip
 
 test:
   imports:
     - backports
     - backports.ssl_match_hostname
 
+outputs:
+  - name: ssl_match_hostname
+    build:
+      script: python -m pip install --no-deps --ignore-installed .
+    requirements:
+      - python
+      - backports
+      - ipaddress
+
+  - name: backports.ssl_match_hostname
+    requirements:
+      - python
+      - {{ pin_subpackage('ssl_match_hostname', exact=True) }}
+
+
 about:
   home: http://bitbucket.org/brandon/backports.ssl_match_hostname
   license:  PSF 2
+  license_family: PSF
   license_file: backports/ssl_match_hostname/LICENSE.txt
```

Comment:
Does anyone know how this works with split packages? Do they all get a copy or just one gets it?

cc @msarahan